packages/btcindexer/src/btcindexer.ts

```diff
@@ -325,7 +349,10 @@ export class Indexer implements Storage {
 				const txIndex = block.transactions.findIndex((tx) => tx.getId() === txId);
 
 				if (txIndex === -1) {
-					console.warn(`Minting: Could not find TX ${txId} within its block. Skipping.`);
+					console.warn({
+						msg: "Minting: Could not find TX within its block, skipping.",
+						txId,
+					});
```

Comment:
Add todo and create a new task - we should add a proper status to those transactions (eg: "dangling")